api/bolt/migrator/migrate_dbversion24.go

```diff
@@ -0,0 +1,12 @@
+package migrator
+
+func (m *Migrator) updateSettingsToDBVersion25() error {
+	legacySettings, err := m.settingsService.Settings()
+	if err != nil {
+		return err
+	}
+
```

Comment:
do we also need to include other db updates to make db version 25 consistent across 1.24 and 2.0:

```
if legacySettings.TemplatesURL == "" {
	legacySettings.TemplatesURL = portainer.DefaultTemplatesURL
}

legacySettings.UserSessionTimeout = portainer.DefaultUserSessionTimeout
```

Reply:
@yi-portainer I'm not sure about this one. Is it decided at the project level to bring them and related features? If yes it should be a separate PR maybe.